### DISCUSSION TITLE
How to use interaction based react-aria hooks with react-aria-components?

### DISCUSSION THREAD

**Discussion by sudo-subg** [adobe/react-spectrum#6260] [Q&A] [answered]
Hey,
I'm working on a notion style slash command menu for a rich text editor using Popover and ListBox components.

While opening the popover and controlling the selected item state with up/down keys when the popover is open is straightforward, I'm having trouble changing selection based on hovering over ListBoxItem component using the useHover hook. The useHover hook is not registering hover events on the list items.

```tsx
function AriaListBoxItem(props: ListBoxItemProps ) {
  let { hoverProps } = useHover({ onHoverStart() {
    console.log(`hovered on item ${props.id}`);
    },
  })

  return (
    <ListBoxItem {...hoverProps} {...props} />
  );
}
```

Is this because of the use of the same hook under the hood, or is there another reason? If there is any other better way then do lemme know :smile:.

Thanks!

**Comment by reidbarber** [comment, marked as answer]
Hover event handlers (`onHoverStart`, `onHoverChange`, `onHoverEnd`) were added to `ListboxItem` via https://github.com/adobe/react-spectrum/pull/5777 (and other collection items via https://github.com/adobe/react-spectrum/pull/6255). Those will be part of the upcoming release, or you can use them now in the nightlies. You won't need to use `useHover` when that becomes available since it is used under the hood.

**Reply by sudo-subg** [reply]
Awesome!! Thanks for the heads-up :+1:

**Reply by upender7270** [reply]
is this released?

**Reply by snowystinger** [reply]
What are you having an issue with in the latest release? Can you provide a codesandbox or stackblitz demonstrating the issue?

**Reply by upender7270** [reply]
I just want to use the `onHoverStart` and `onHoverEnd` events to display the menu on hover on listBox item

**Reply by snowystinger** [reply]
Can you provide a codesandbox or stackblitz demonstrating the issue?

In general, we don't recommend showing/hiding anything on hover, this is not accessible to keyboard users or touch users or assistive technologies. Nobody but a mouse user can activate hover.
